server/views/pages/approve/cases.njk

```diff
@@ -17,6 +17,9 @@
     {% elseif case.isDueForEarlyRelease and approvalNeededView %}
         {% set releaseDate = '<span class="urgent-highlight">' + case.releaseDate
             + '</span><span class="urgent-highlight urgent-highlight-message">Early release</span>' %}
+    {% elseif case.kind == 'HDC' and approvalNeededView %}
+        {% set releaseDate = '<span class="urgent-highlight">' + case.releaseDate
+            + '</span><span class="urgent-highlight urgent-highlight-message">HDC release</span>' %}
```

Comment:
mmh.. not sure if we need to do it for both approval needed and the approval given view

Reply:
Sorry I'm confused - assume `approvalNeededView` is the view we do want to show it in? and therefore that's what the above is doing? Is approval given view for `'/pdf-print'`? I think I need to keep `and approvalNeededView` 😅

Reply:
Design confirmed it's needed for both approval needed and recently approved tab views so updated for this